.changeset/tiny-lamps-melt.md

```diff
@@ -1,5 +1,5 @@
 ---
-"@lion/calendar": patch
+'@lion/calendar': patch
```

Comment:
Shouldn't we remove all open changesets?

And have 1 changeset for this change?

Reply:
yeah, but let's do this when we are actually closer to a release